astroid/const.py

```diff
@@ -17,3 +18,5 @@ class Context(enum.Enum):
 Load = Context.Load  # pylint: disable=invalid-name
 Store = Context.Store  # pylint: disable=invalid-name
 Del = Context.Del  # pylint: disable=invalid-name
+
+BUILTINS = builtins.__name__  # Could be just 'builtins' ?
```

Comment:
In theory it shouldn't matter, but I kinda understand why `builtins.__name__` was chosen. That way you know at first glance it's the name of the builtins module and don't need an additional comment explaining it.